### DISCUSSION TITLE
Chataigne module

### DISCUSSION THREAD

**Discussion by cpvalente** [cpvalente/ontime#851] [Discussion]
It has been suggested by several users that a chataigne integration could be useful.

I am hoping to gauge the community interest on this and write a specification.

In what workflows does chataigne and ontime work together?
What would an eventual integration allow you to do?

Note to self, the repository of modules is here
https://github.com/benkuper/Chataigne-community-modules/blob/main/modules.json

**Comment by je71175** [comment]
I think this is a great idea. I've been able to achieve most of my needs via OSC, so I guess that would be a minimum. Perhaps other things not yet implemented in OSC/api for example be able to integrate the ability to load / refresh Google sheets load new worksheet etc via the module. Will think some more on this but exciting ideas

**Comment by alex-Arc** [comment]
Looks very possible, and seams to also support WebSockets that might be easier for us to maintain as it will be almost the same as the companion module
https://bkuperberg.gitbook.io/chataigne-docs/modules/making-your-own-module

**Comment by cpvalente** [comment]
chataigne module is live
for question or suggestions on this, please follow the module repository
https://github.com/alex-Arc/Ontime-Chataigne-module
